### PR TITLE
Add buildship offline mode preference

### LINKED ISSUE #1157: Add buildship offline mode preference
Related to #901

### PR DESCRIPTION
**Pull request by snjeza**
Fixes #1157 
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/1308

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,10 +119,11 @@ The following settings are supported:
 * `java.codeGeneration.toString.listArrayContents`: List contents of arrays instead of using native toString(). Defaults to `true`.
 * `java.codeGeneration.toString.limitElements`: Limit number of items in arrays/collections/maps to list, if 0 then list all. Defaults to `0`.
 * `java.selectionRange.enabled`: Enable/disable Smart Selection support for Java. Disabling this option will not affect the VS Code built-in word-based and bracket-based smart selection.
-
-New in 0.54.2:
 * `java.showBuildStatusOnStart.enabled`: Automatically show build status on startup. Defaults to `false`.
+* "Organize imports" is triggered when pasting code into a java file with `Ctrl+Shift+v` (`Cmd+Shift+v` on Mac).
```
Comment (fbricon) [inline README.md]: unnecessary. This is not a preference and it's already listed above, in the features section

File: README.md [hunk 2]
```diff
@@ -119,10 +119,11 @@ The following settings are supported:
...
 
+New in 0.55.0:
```
Comment (fbricon) [inline README.md]: 0.56.0

File: README.md (hunk 1 above)
Comment (fbricon) [reply README.md]: please remove that line

**Comment by snjeza** [comment]
@fbricon I have updated the PR.

**Comment by fbricon** [comment]
Thanks@snjeza

**Pull request merged**